dom/src/main/java/org/apache/james/mime4j/message/AbstractHeader.java

```diff
@@ -86,6 +86,17 @@ public void addField(Field field) {
         return Collections.unmodifiableList(fields);
     }
 
+    /**
+     * Gets the fields of this header. The returned map will not be
```

Comment:
what about using `{@inheritdoc}` ?

Reply:
Thanks for the tip. 

I decided to be consistent with the javadoc already written that do not use `{@inheritdoc}`

However I would support a switch in another PR at the scope of the project.